styles.css

```diff
@@ -1,543 +1,543 @@
+  }
+  .get{
+    width: 34vw;
+    height: 8vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 4vw;
+    line-height: 8vw;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396DF2;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    border: #396DF2 solid 1.5px;
+    background-color: white;
+    border-radius: 8px;
+    margin-left: 3vw;
+    padding: 1vw 2vw 1vw 2vw;
+  }
+  .lang{
+    width:63.2vw;
+    height: 6vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 5.33vw;
+    line-height: 6vw;
+    display: flex;
+    align-items: center;
+    color: #000000;
+    margin-left: 1vw;
+    padding-top: 2vw;
+  }
+  .vec{
+    width: 5vw;
+    height: 3vw;
+    padding-top: 2vw;
+  }
+  .ic{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+  }
+  .down{
+    display: flex;
+    flex-direction: column;
+    width: 78.67vw;
+    height: 6vw;
+    gap: 10vw;
+    }
+    .jsfile{
+      width: 75vw;
+      height: 15vw;
+      background: #F7F7F9;
+      border-radius: 8px;
+      flex: none;
+      order: 0;
+      flex-grow: 0;
+      display: flex;
+      flex-direction: row;
+      flex-wrap: nowrap;
+    }
+    .js{
+      width: 75vw;
+      height: 15vw;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 4vw;
+      line-height: 5vw;
+      display: flex;
+      align-items: center;
+      color: #253858;
+      text-indent: 18vw;
+      padding-top:  2vw;
+      background: url("js.png");
+      background-size: 12vw 12vw;
+      background-repeat: no-repeat;
+      background-position: 4vw 2vw;
+      border: 1px solid #FFFFFF;
+    }
+    .html5{
+      width: 75vw;
+      height: 15vw;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 4vw;
+      line-height: 5vw;
+      display: flex;
+      align-items: center;
+      color: #253858;
+      text-indent: 18vw;
+      padding-top:  2vw;
+      background: url("html5.png");
+      background-size: 12vw 12vw;
+      background-repeat: no-repeat;
+      background-position: 4vw 2vw;
+      border: 1px solid #FFFFFF;
+    }
+    .css3{
+      width: 75vw;
+      height: 15vw;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 4vw;
+      line-height: 5vw;
+      display: flex;
+      align-items: center;
+      color: #253858;
+      text-indent: 18vw;
+      padding-top:  2vw;
+      background: url("css3.png");
+      background-size: 12vw 12vw;
+      background-repeat: no-repeat;
+      background-position: 4vw 2vw;
+      border: 1px solid #FFFFFF;
+    }
+    .line{
+      width: 78.67vw;
+      height: 1vw;
+      margin-left: 1vw;
+      margin-right: 1vw;
+      border: #DFE1E6;

```

Comment:
- [ ] Upon careful examination, I have observed that the initial project you created, specifically the "Portfolio: setup and mobile first," has been intentionally removed. Instead, a new CSS file has been introduced in a separate branch, containing only the CSS code for the "Portfolio: finish mobile version" task. To ensure a streamlined development process, it is highly recommended to consolidate all CSS code into a unified `styles.css` file. Having separate CSS files for each task can potentially introduce significant complexities when merging them into the main branch of your repository. Therefore, I strongly advise consolidating the CSS code into a single `styles.css` file to proactively address any future challenges.